adapters/AppLovin/AppLovinAdapter/Bidding/GADMRTBAdapterAppLovinInterstitialRenderer.m

```diff
@@ -73,6 +73,7 @@ - (void)loadAd {
 
   // Create interstitial object.
   _interstitialAd = [[ALInterstitialAd alloc] initWithSdk:_SDK];
+  [_interstitialAd setExtraInfoForKey:@"google_watermark" value:_adConfiguration.watermark];
```

Comment:
@Vedant-Mehta39 
As `_adConfiguration.watermark` is `NSData` type, can you confirm type conversion isn't needed before doing `setExtraInfoForKey`?

Reply:
Yes, type conversion isn't needed